AdaSupport/embed
# FND-1181: Be able to point to different clusters/bots when testing

Next stage of embed improvements
Steps to test:
1. Add values in the script tag like below:
```
data-handle="baba"
data-domain="ada-dev"
```

2. Run `yarn deploy-test` the hosted embed bot should point to `baba.ada-dev.support`.

Todo:
- Confirm backwards compatibility and general cleanup
---
~~- [ ] PR is linked to a Jira card has `impact` field filled in~~
~~- [ ] Changes have gone through design review if required~~
~~- [ ] Deployment plan and/or bridge code description is included in PR if required~~
~~- [ ] Changes are CATO/WCAG compliant and support IE9+~~
~~- [ ] A [Release Note](https://www.notion.so/adasupport/Creating-Release-Notes-36906dfa8a2b4f10a31dc23b8d46681a) has been drafted and published after merging to master~~
